Makefile

```diff
@@ -120,7 +120,6 @@ help: ## Print help for targets with comments.
 	@printf "For more targets and info see the comments in the Makefile.\n\n"
 	@grep -E '^[[:alnum:]._-]+:.*?## .*$$' Makefile | sort | \
 		awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-15s\033[0m %s\n", $$1, $$2}'
-
```

Comment:
This seems unrelated?

Reply:
Sorry! I was planning to fix the logic, but I realized it's fine as is. Fixed!